### ISSUE TITLE
Subtitles not remuxed

### ISSUE THREAD

**Issue by Sharc99** [rigaya/NVEnc#252]
Source is an .m2ts file with PGS subtitles.
--sub-copy seems not to work for remuxing into .ts, .m2ts, .mp4 containers.  It works for .mkv only.
Is this a bug, or do I miss something?

**Comment by ValZapod** [comment]
PGS subtitles are not really supported by ffmpeg (it is oversimplified statement and code is really there, but...)

**Comment by Sharc99** [comment]
Actually it works perfectly with ffmpeg.exe -map 0 -c:s copy -c:v h264 nvenc ........
So I assumed it should be working with NVEncC64.exe --sub-copy ........ as well .

**Comment by Sharc99** [comment]
Issue closed by mistake

**Comment by rigaya** [comment]
- m2ts containers
  I have updated ffmpeg dlls (such as libavformat modules) in [NVEnc 5.13](https://github.com/rigaya/NVEnc/releases/tag/5.13), which will add support for muxing PGS subtitles into m2ts containers.

- mp4 containers
  The mp4 standard does not have support for PGS subs, so it cannot be remuxed into mp4 containers. You will also get error message below in ffmpeg.
[mp4 @ 0000020ea0cfc380] Could not find tag for codec hdmv_pgs_subtitle in stream #1, codec not currently supported in container

- ts containers
  Seems to be not supported yet in libavformat. I got following warning in ffmpeg 4.3.
  [mpegts @ 000001f0cf5ec380] Stream 1, codec hdmv_pgs_subtitle, is muxed as a private data stream and may not be recognized upon reading.
